Gulpfile.js

```diff
@@ -19,6 +21,11 @@ const files = {
     js: './src',
     static: ['./www/**/*', './config.xml', './electron.js', '!./www/**/*.scss'],
     sass: './styles/*.scss',
+    androidApk: './platforms/android/build/outputs/apk',
+    unalignedApk: './platforms/android/build/outputs/apk/android-release-unaligned.apk',
+    alignedApk: './platforms/android/build/outputs/apk/android-release-aligned.apk',
+    zipAligned: '/Users/nuru/Library/Android/sdk/build-tools/23.0.3/zipalign', // Note the path to the zipalign on your pc
```

Comment:
@binarytom inorder to have ANDROID_SDK_ROOT avail in gutil.env , we need to hard code it in package.json don't we? Or is there any other way we can fetch it?